### PR TITLE
MS1 - Layout Using Bootstrap 5.0.x

### PR DESCRIPTION
**Pull request by marijanbrvar**
#Newsweek clone page
- Created elements for the header
- Creating elements for Footer
- Adding  content for navigation and footer part
- Laying elements for all tree screens using Scss
- Styling elements

### CODE REVIEW

File: index.html
Comment (teekaytech) [inline index.html]: This Newsweek logo should be linked. i.e. it should be inside the `<a>` tag.

File: index.html
Comment (teekaytech) [inline index.html]: **In mobile and tablet screen sizes, the menu should hide, and show a hamburger button**

File: index.html
Comment (teekaytech) [inline index.html]: The Newsweek logo should be linked.
-  Instead of duplicating the same SVG in different places, you can easily extract the image to a separate file, then reference the image in both the header and footer, using the image tag.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__This is an awesome starting point for this project :clap:__

- Your linters are running successfully :100: 
- Your PR title and description looks great :ok: 
- Your readme is awesome :+1: 

__However, some changes are required before this milestone can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] The `MIT license link` on your readme is broken. Kindly fix it.
- [x] The implementation requirement of this project specifies that you use Bootstrap CSS classes everywhere for styling (for example Grid layout and Flex).
- At the moment, you're having some CSS rules in your header.scss, and footer.scss files (like black color, white color, padding, margin, etc) that can be implemented with bootstrap.
- Kindly ensure that you use bootstrap for all stylings where bootstrap classes are available.
- I understand that this might be a difficult task in the beginning. However, researching and knowing these bootstrap class names will help you in the long run.
- [x] Kindly check the inline comments below for other required changes.



* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

File: index.html
Comment (marijanbrvar) [reply index.html]: Newsweek logo is in Vector format and has a different fill (e.g. in the header, it has a white fill and in the footer is red) to target SVG fill property embedded in IMG resource is beyond my knowledge, so I think that easiest way to change fill colour is to embed SVG in the page. I f you insist on this requirement, I will make two copies of the same vector image and link it over the IMG element. Please advise

File: index.html
Comment (marijanbrvar) [reply index.html]: It is hidden on medium screens, the Hamburger button is missing, and I am working on it!

**Review by SpaYco — APPROVED** [review]
# Project Approved ✔️ 

thank you for making the required changes, your project is now approved!

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**
